### PR TITLE
Use fetch to print random greetings on home page

### PR DESCRIPTION
**Pull request by kierabr1**
I modified DataServlet.java, script.js, and index.html to print random greetings on the home page by using fetch().

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,10 @@ function addRandomFunFact() {
   const funfactContainer = document.getElementById('funfact-container');
   funfactContainer.innerText = funfact;
 }
+
+function getRandomQuoteUsingArrowFunctions() {
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js]: I would just call the function getRandomQuotes. The fact that you use lambdas is an implementation details of the function and does not impact the caller of this function.

File: walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingAsyncAwait()">Fetch a quote!</button>
```
Comment (sbahaud) [inline walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html]: Remove this change. This is one of the examples and you probably accidentally edited it.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -23,10 +25,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: Add period at the end of the sentence. Make comments completed sentences

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -23,10 +25,21 @@
...
+  private List<String> quotes;
 
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Welcome");
+    quotes.add("Bienvenidos");
+    quotes.add("Bonjour");
+    quotes.add("Aloha");
+    quotes.add("Kon'nichiwa");
+  }
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: nit: Add a blank line after this. 1 blank line between methods.

**Review by sbahaud — COMMENTED** [review]
Left some minor comments

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/script.js]: done

File: walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html (hunk 2 above)
Comment (kierabr1) [reply walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html]: done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (kierabr1) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (kierabr1) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done

**Review by sbahaud — APPROVED** [review]
LGTM

**Pull request merged**